entity-framework/core/miscellaneous/1x-2x-upgrade.md

```diff
@@ -89,19 +89,19 @@ The SQL Server and SQLite providers are shipped by the EF team and 2.0 versions
 
 Note: these changes should not impact most application code.
 
-The event IDs for messages sent to an [ILogger](https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging.Abstractions/ILogger.cs) have changed in 2.0. The event IDs are now unique across EF Core code. These messages now also follow the standard pattern for structured logging used by, for example, MVC.
+The event IDs for messages sent to an [ILogger](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.ilogger) have changed in 2.0. The event IDs are now unique across EF Core code. These messages now also follow the standard pattern for structured logging used by, for example, MVC.
 
-Logger categories have also changed. There is now a well-known set of categories accessed through [DbLoggerCategory](https://github.com/aspnet/EntityFramework/blob/dev/src/EFCore/DbLoggerCategory.cs).
+Logger categories have also changed. There is now a well-known set of categories accessed through [DbLoggerCategory](https://github.com/aspnet/EntityFramework/blob/master/src/EFCore/DbLoggerCategory.cs).
```

Comment:
If this refers to 2.1 release of EF Core, we have API docs for 2.1 release.
If there are no API docs for us then it should reference tag rather than master branch.

Reply:
Also aspnet/EntityFrameworkCore we renamed our repo loooong time ago ;)

Reply:
> If this refers to 2.1 release of EF Core, we have API docs for 2.1 release.

The "well-known set of categories" mentioned here aren't listed in the API docs, that's why this is the only link I left pointing to the source. But updated the version to rel/2.0.0 and the repo name. Will merge, we can fix if you feel strongly about this for some reason.